### PR TITLE
CB-3170 - NPE fixed during termination in CloudFileSystemViewBuilder

### PR DESCRIPTION
**Pull request by attilapalfi92**

### CODE REVIEW

File: core/src/main/java/com/sequenceiq/cloudbreak/converter/spi/StackToCloudStackConverter.java [hunk 1]
```diff
@@ -202,7 +202,7 @@ InstanceTemplate buildInstanceTemplate(Template template, String name, Long priv
         List<Group> groups = new ArrayList<>();
         Cluster cluster = stack.getCluster();
         if (cluster != null && cluster.getBlueprint() != null) {
```
Comment (lnardai) [inline core/src/main/java/com/sequenceiq/cloudbreak/converter/spi/StackToCloudStackConverter.java]: If we are using the BlueprintText, probably we won't need to getBlueprint nullcheck

File: core/src/main/java/com/sequenceiq/cloudbreak/converter/spi/CloudFileSystemViewBuilder.java [hunk 2]
```diff
@@ -25,7 +25,8 @@
         Optional<CloudFileSystemView> fileSystemView;
         if (fileSystem != null) {
             SpiFileSystem spiFileSystem = fileSystemConverter.fileSystemToSpi(fileSystem);
-            if (componentsByHostGroup.get(instanceGroup.getGroupName()).contains(KnoxRoles.IDBROKER)) {
+            Set<String> components = componentsByHostGroup.get(instanceGroup.getGroupName());
+            if (components != null && components.contains(KnoxRoles.IDBROKER)) {
                 fileSystemView = spiFileSystem.getCloudFileSystems().stream()
                         .filter(cloudFileSystemView -> cloudFileSystemView.getCloudIdentityType().equals(CloudIdentityType.ID_BROKER))
```
Comment (lnardai) [inline core/src/main/java/com/sequenceiq/cloudbreak/converter/spi/CloudFileSystemViewBuilder.java]: Can you swap the equals to have constants in front?

**Comment by keyki** [comment]
@attilapalfi92 Talked with @lnardai, can you please change the base version to rc-2.12?

**Comment by attilapalfi92** [comment]
@keyki done

**Pull request merged**